app/webpacker/components/Panel/pages/BannedCompetitorsPage/BannedCompetitors.jsx

```diff
@@ -20,6 +20,8 @@ export default function BannedCompetitors({
             <Table.HeaderCell width={5}>User</Table.HeaderCell>
             <Table.HeaderCell width={2}>Start date</Table.HeaderCell>
             <Table.HeaderCell width={2}>End date</Table.HeaderCell>
+            <Table.HeaderCell width={2}>Reason</Table.HeaderCell>
+            <Table.HeaderCell width={2}>Scope</Table.HeaderCell>
```

Comment:
SemUI widths should ideally add up to 16

Reply:
Oh I didn't knew it, done.